app/model/application.js

```diff
@@ -362,12 +220,9 @@ ApplicationSchema.methods.pushGithub = function() {
 // Returns saved application including 'errors' always
 ApplicationSchema.methods.syncGithub = function() {
   let application = this;
-  const repoName = application.github.name;
-  const fullName = application.github.fullName;
-  app.log.silly(`application syncGithub called for ${fullName}`);
 
   return application.fetchGithub()
-  .then(application => application.pushGithub()); // Push doesn't save anything
+  .then(application => application.pushGithub); // Push doesn't save anything
```

Comment:
that's my bad, I was reading about promises and returning async functions and what not. Must not have changed this back from my time messing around